javascript-carousel/styles.css

```diff
@@ -0,0 +1,74 @@
+.pikachuAndArrows,
+.bulbasaurAndArrows,
+.charmanderAndArrows,
+.squirtleAndArrows,
+.jigglypuffAndArrows {
+  display: flex;
+  justify-content: space-around;
+
+
+}
+.hidden {
+  display: none;
+}
+
+
+img {
+    width: 700px;
+}
+
+.pikachuDot1,
+.pikachuDot2,
+.pikachuDot3,
+.pikachuDot4,
+.pikachuDot5,
+.bulbasaurDot1,
+.bulbasaurDot2,
+.bulbasaurDot3,
+.bulbasaurDot4,
+.bulbasaurDot5,
+.charmanderDot1,
+.charmanderDot2,
+.charmanderDot3,
+.charmanderDot4,
+.charmanderDot5,
+.squirtleDot1,
+.squirtleDot2,
+.squirtleDot3,
+.squirtleDot4,
+.squirtleDot5,
+.jigglypuffDot1,
+.jigglypuffDot2,
+.jigglypuffDot3,
+.jigglypuffDot4,
+.jigglypuffDot5 {
+  margin: 100px 10px 0 10px;
+  font-size: 50px;
+  background-color: white;
+  border: none;
+  cursor: pointer;
+}
```

Comment:
You can probably get away with using a single class for these style rules, instead of 25 unique ones.